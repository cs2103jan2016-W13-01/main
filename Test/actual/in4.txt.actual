Task added successfully
1. incomplete; eat dinner; -; -; -
Task edited: eat dinner
1. incomplete; eat dinner; -; -; -
Invalid command. Please try again.
1. incomplete; eat dinner; -; -; -
Task added successfully
1. incomplete; eat dinner; -; -; -
2. incomplete; play game; -; -; -
Task edited: play game
1. incomplete; eat dinner; -; -; -
2. incomplete; play game; -; -; -
