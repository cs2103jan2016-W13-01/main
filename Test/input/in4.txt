add eat dinner
edit 1 eat lunch
edit 2
add play game
edit 2 study