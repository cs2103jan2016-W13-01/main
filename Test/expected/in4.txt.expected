Task added successfully: eat dinner  Once
1. incomplete; eat dinner; -; -; Once
Task edited: eat dinner  Once
1. incomplete; eat dinner; -; -; Once